src/UI/Factory.php

```diff
@@ -538,4 +538,33 @@ public function link();
 	 */
 	public function dropdown();
 
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *     Breadcrumbs is a type of secondary navigation scheme. It eases the
+	 *     user's navigation back to higher items in hierarchical (or prior items
+	 *     in chronological) structures.
+	 *     Breadcrumbs also serve as an effective visual aid indicating the
+	 *     user's location on a website.
+	 *     The term is based on the story of Hansel and Gretel and is well established,
+	 *     so we will stick to it, although in the context of ILIAS "Ariadne's Path"
+	 *     would probably be more appropriate.
+	 *   composition: >
+	 *     Breadcrumbs-entries are rendered as horizontally arranged UI Links
+	 *     with a seperator in-between.
+	 *   effect: >
+	 *     Clicking on an entry will get the user to the respective location.
+	 *
+	 * context: >
+	 *
+	 * rules:
+	 *   usage:
+	 *     1: Crumbs MUST trigger navigation.
```

Comment:
Jour Fixe: "Crumbs must trigger navigation to other resources of the system." (similar to Link definition)